src/ucp/core/ucp_context.c

```diff
@@ -100,6 +100,10 @@ static ucs_config_field_t ucp_config_table[] = {
    "the eager_zcopy protocol",
    ucs_offsetof(ucp_config_t, ctx.rndv_perf_diff), UCS_CONFIG_TYPE_DOUBLE},
 
+  {"MAX_RNDV_LANES", "1",
+   "Set max multirail-get rendezvous lane numbers",
```

Comment:
Should we get rid of `multirail` word here as well?

Reply:
multi-rail is still here

Reply:
@brminich @yosefe 
"Set max rendezvous-get lane numbers"
will work?

Reply:
"Maximal number of devices on which a rendezvous operation may be executed in parallel."

Reply:
updated